polyfills/Symbol/config.json

```diff
@@ -8,7 +8,7 @@
 		"safari": "<9",
 		"chrome": "< 50",
 		"firefox": "< 40",
-		"ios_saf": "<9",
+		"ios_saf": "<10",
```

Comment:
In production the config is currently set to `*`, that is why ios saf 10 is getting symbol :-)